src/matching/Generic_vs_generic.ml

```diff
@@ -167,6 +167,12 @@ let should_match_call = function
   | G.IncrDecr _ ->
       false
 
+let m_id_string case_insensitive =
+  let open String in
```

Comment:
Not a huge fan of open or of local open.
Qualify is usually clearer, unless it really adds lots of text, but in that case we usually
prefer module aliasing with shorter names for the alias (e.g., module G = AST_generic).